src/styles/main.scss

```diff
@@ -0,0 +1,75 @@
+$item-bg-color: #eee;
+$calendar-item-size: 100px;
+$calendar-gap: 1px;
+$calendar-item-width: $calendar-item-size + $calendar-gap;
```

Comment:
The calculation of `$calendar-item-width` includes a `1px` gap, which might not be the desired behavior for the width of calendar items. If the gap is intended only between items, the width should not include the gap size.